safe/gui/widgets/dock.py

```diff
@@ -1742,7 +1752,14 @@ def layer_changed(self, layer):
         try:
             keywords = self.keyword_io.read_keywords(layer)
 
-            if keywords.get('layer_purpose') == 'impact':
+            # list of layer purpose to show impact report
+            impacted_layer = [
+                'impact',
+                'analysis',
+                'exposure_breakdown',
+            ]
```

Comment:
So if the user is running a population raster exposure, he will not have any report for now ? Because population raster, the impact layer is an aggregate_hazard. Correct me if I didn't understand correctly